src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.PortedTypes.cs

```diff
@@ -25,7 +25,8 @@ internal sealed partial class IOperationClassWriter
             "ISizeOfOperation",
             "IOmittedArgumentOperation",
             "IDiscardOperation",
-            "IPlaceholderOperation"
+            "IPlaceholderOperation",
+            "IBlockOperation"
```

Comment:
Consider always adding the trailing comma here, makes the commit by commit diffs a tad simpler to process.

Reply:
I'll try to do this going forward, but I'm not going to go back and amend the 24 commits I have ready for the next PR.